content/manager/bootstrapping.md

```diff
@@ -165,6 +165,153 @@ Services:
+    | openstack_tenant_name           | OpenStack |             | Tenant name to use when accessing openstack API.                                                                                                                                                                                                                         |
+    | openstack_region                | OpenStack |             | Region to use when accessing openstack API.                                                                                                                                                                                                                              |
+    | manager_names_and_ips           |           | ✓           | You should include the manager's public IP here, as well as any DNS names you want to assign to the manager (comma separated). Internal IPs on your platform will be added automatically.  e.g. 192.0.2.54,mycloudify.example.com,mycloudify.dev.example.com,192.0.2.100 |
+    | broker_names_and_ips            |           | ✓           | As above                                                                                                                                                                                                                                                                 |
+    | new_manager_username            |           | ✓           | New username for the cloudify manager.                                                                                                                                                                                                                                   |
+    | new_manager_password            |           | ✓           | New password for the cloudify manager.                                                                                                                                                                                                                                   |
+    | new_broker_username             |           | ✓           | New username for the message broker.                                                                                                                                                                                                                                     |
+    | new_broker_password             |           | ✓           | New password for the message broker.                                                                                                                                                                                                                                     |
+
+    {{% gsNote %}}
+    It is possible to use the contents of an inputs file (if formatted as JSON) directly by pasting them into the "raw" field.
```

Comment:
I think you're right, but the web UI just won't let you create until you've fixed that, so do we need to point it out here too?